examples/nffPktgen/generator/generator.go

```diff
@@ -116,7 +116,7 @@ func generateIPv4(pkt *packet.Packet, config *PacketConfig, rnd *rand.Rand) {
 	}
 	copyDataFunc(dataConfig, size, rnd, pkt.Data)
 	fillEtherHdr(pkt, l2)
-	fillIPv4Hdr(pkt, l3)
+	FillIPv4Hdr(pkt, l3)
```

Comment:
It is inconsistent to have some functions as exported and the same functions as unexported.

Reply:
Ok